# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/Stencil.cpp
# Commit: ca60758cd534
# Full Hash: ca60758cd534c0f3bb04a7acda15eb41a67e7389
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 12: Use TaggedParserAtomIndex in ObjLiteralStencil. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96919
# ==============================================================================

diff -r 8af30c6d9bb9 -r ca60758cd534 js/src/frontend/Stencil.cpp
--- a/js/src/frontend/Stencil.cpp	Sat Nov 14 00:03:36 2020 +0000
+++ b/js/src/frontend/Stencil.cpp	Sat Nov 14 00:03:45 2020 +0000
@@ -932,9 +932,9 @@
 
 #if defined(DEBUG) || defined(JS_JITSPEW)
 
-void DumpTaggedParserAtomIndex(js::JSONPrinter& json,
-                               TaggedParserAtomIndex taggedIndex,
-                               CompilationStencil* compilationStencil) {
+void frontend::DumpTaggedParserAtomIndex(
+    js::JSONPrinter& json, TaggedParserAtomIndex taggedIndex,
+    CompilationStencil* compilationStencil) {
   if (taggedIndex.isParserAtomIndex()) {
     json.property("tag", "AtomIndex");
     auto index = taggedIndex.toParserAtomIndex();
@@ -1642,7 +1642,7 @@
 
   json.beginListProperty("objLiteralData");
   for (auto& data : objLiteralData) {
-    data.dump(json);
+    data.dump(json, this);
   }
   json.endList();
 